Makefile

```diff
@@ -157,19 +157,18 @@ install-glide:
 install: install-glide
 	glide install
 
-.PHONY: install-go-bindata
-install-go-bindata:
-	go get github.com/jteeuwen/go-bindata/...
-	go get github.com/elazarl/go-bindata-assetfs/...
+.PHONY: install-statik
+install-statik:
+	go get github.com/rakyll/statik
 
 .PHONY: build-examples
-build-examples: install-go-bindata
-	(cd ./examples/hotrod/services/frontend/ && go-bindata-assetfs -pkg frontend web_assets/...)
-	rm ./examples/hotrod/services/frontend/bindata.go
-	CGO_ENABLED=0 GOOS=linux installsuffix=cgo go build -o ./examples/hotrod/hotrod-linux ./examples/hotrod/main.go
+build-examples: install-statik
+	(cd examples/hotrod/services/frontend/ && statik -f --src web_assets)
+	CGO_ENABLED=0 installsuffix=cgo go build -o ./examples/hotrod/hotrod-$(GOOS) ./examples/hotrod/main.go
 
 .PHONE: docker-hotrod
-docker-hotrod: build-examples
+docker-hotrod:
+	GOOS=linux $(MAKE) build-examples
```

Comment:
This sounds a bit odd. What was wrong with the previous approach?

Reply:
previously build-examples target didn't work for multiple architectures. Now it is in line with other `build-{binary}` targets.

Reply:
I see. And because it's Docker, `GOOS=linux` is appropriate. Makes sense, thanks for the explanation.